tests/test_core.py

```diff
@@ -6127,6 +6157,10 @@ def do_autodebug(filename):
   @no_fastcomp('autodebugging wasm is only supported in the wasm backend')
   @with_env_modify({'EMCC_AUTODEBUG': '1'})
   def test_autodebug_wasm(self):
+    # Autodebug does not work with too much shadow memory
```

Comment:
hmm, why not?

Reply:
Added a comment explaining why. Basically the more WASM linear memory we have, the more memory autodebug uses and it causes the JS engine to OOM with too much shadow memory.